src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -162,14 +270,27 @@ const UserVocabularyProcessScreen = ({ navigation }: UserVocabularyProcessScreen
         />
         <StyledHintText>{maxPictureUpload}</StyledHintText>
         {hasImageErrorMessage && <ContentError>{getLabels().userVocabulary.creation.imageErrorMessage}</ContentError>}
-
-        <AddAudioButton
-          onPress={() => null}
-          label={addAudio}
-          buttonTheme={BUTTONS_THEME.text}
-          iconLeft={MicrophoneCircleIcon}
-          iconSize={theme.spacingsPlain.xl}
-        />
+        {recordingPath ? (
+          <AudioContainer style={{ opacity: fadeAnim }}>
+            <>
+              <AudioText>{recordingPath.substring(recordingPath.lastIndexOf('/') + 1).toUpperCase()}</AudioText>
+              <DeleteContainer onPress={() => deleteRecording()} testID='delete-audio-recording'>
+                <CloseCircleIconBlue width={theme.spacingsPlain.lg} height={theme.spacingsPlain.lg} />
+              </DeleteContainer>
+              <PlayContainer onPress={() => onStartPlay()} isActive={isPlaying} testID='play-audio-recording'>
+                <VolumeUpCircleIcon width={theme.spacingsPlain.lg} height={theme.spacingsPlain.lg} />
+              </PlayContainer>
+            </>
+          </AudioContainer>
+        ) : (
+          <AddAudioButton
+            onPress={() => setShowAudioRecordOverlay(true)}
+            label={addAudio}
+            buttonTheme={BUTTONS_THEME.text}
+            iconLeft={MicrophoneCircleIcon}
+            iconSize={theme.spacingsPlain.xl}
+          />
+        )}
```

Comment:
But this not the AudioRecorder its just a component to play or delete an audiofile

Reply:
moved to separate component. thx for all the hints. I removed the fadeAnimation, since its breaking it separate component. If this is mandatory lets create a separate ticket for a reusable animation container or sth